src/Microsoft.DotNet.Build.Tasks/Targets/partialfacades.targets

```diff
@@ -134,11 +134,11 @@
     <Error Text="No single matching contract found." Condition="'@(ResolvedMatchingContract->Count())' != '1'" />
 
     <PropertyGroup>
-      <GenFacadesArgs>-partialFacadeAssemblyPath:$(GenFacadesInputAssembly)</GenFacadesArgs>
-      <GenFacadesArgs>$(GenFacadesArgs) -contracts:%(ResolvedMatchingContract.Identity)</GenFacadesArgs>
-      <GenFacadesArgs>$(GenFacadesArgs) -seeds:@(GenFacadesSeeds, ';')</GenFacadesArgs>
-      <GenFacadesArgs>$(GenFacadesArgs) -facadePath:$(GenFacadesOutputPath)</GenFacadesArgs>
-      <GenFacadesArgs Condition="'@(SeedTypePreference)' != ''">$(GenFacadesArgs) -preferSeedType:@(SeedTypePreference->'%(Identity)=%(Assembly)', ',')</GenFacadesArgs>
+      <GenFacadesArgs>-partialFacadeAssemblyPath:&quot;$(GenFacadesInputAssembly)&quot;</GenFacadesArgs>
+      <GenFacadesArgs>$(GenFacadesArgs) -contracts:&quot;%(ResolvedMatchingContract.Identity)&quot;</GenFacadesArgs>
```

Comment:
You don't actually need to use `&quot;` normal quotes `"` work in property groups.